src/r5py/util/jvm.py

```diff
@@ -24,6 +25,14 @@ def start_jvm():
     line and configuration options.
     """
     if not jpype.isJVMStarted():
+        # preload signal handling; this, among other things, prevents some of
+        # the warning messages we have been seeing
+        # (cf. https://stackoverflow.com/questions/
+        #   15790403/what-does-consider-using-jsig-library-mean )
+        LIBJSIG = str(pathlib.Path(jpype.getDefaultJVMPath()).parent / "libjsig.so")
+        os.environ["LD_PRELOAD"] = LIBJSIG  # Linux, Windows
```

Comment:
Can we add tests for these?

Reply:
Not sure how, to be honest. Would you mean testing whether after starting the jvm, these env variables are defined? That would not really test this logic, but more the builtin library `os`. Any suggestions?

Reply:
No I was more thinking, whether we could eg somehow catch the output that the JPype produces and have a simple test that would check that the output. Not sure whether makes sense, but this came to mind

Reply:
Again, not sure how to do that. If we could catch the output, we would have just hidden it long ago. (of course, this is much better, treating the cause and not the symptoms)